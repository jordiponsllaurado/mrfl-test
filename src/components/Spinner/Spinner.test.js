import { render } from '@testing-library/react';
import Spinner from './Spinner';

test('renders Spinner', () => {
  const { container } = render(<Spinner />);
  expect(container).toMatchSnapshot();
});
