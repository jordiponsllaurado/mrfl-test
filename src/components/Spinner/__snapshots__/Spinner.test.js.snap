// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner 1`] = `
<div>
  <div
    class="spinner"
  >
    <div
      class="loader"
    />
  </div>
</div>
`;
